src/general/EnumHelper.cs

```diff
@@ -0,0 +1,34 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+public static class EnumHelper
+{
+    /// <summary>
+    ///   Fetches all CustomAttributes of type T
```

Comment:
I think this and the next method descriptions should say that these fetch the attributes on an *enum* value. At least that's what I understood this to do.